test/units/module_utils/test_vmware.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import sys
+import pytest
+from ansible.module_utils.vmware import connect_to_api, PyVmomi
```

Comment:
Make sure the tests will skip if external dependencies like pyvmomi are not available.

Reply:
Resolved by 
```pyvmomi = pytest.importorskip('PyVmomi')```